packs/data/equipment.db/hallajin-key.json

```diff
@@ -0,0 +1,59 @@
+{
+    "_id": "5rpNqGkPSMkXKv0B",
+    "img": "systems/pf2e/icons/default-icons/equipment.svg",
+    "name": "Hallajin Key",
+    "system": {
+        "baseItem": null,
+        "containerId": null,
+        "description": {
+            "value": "<p><strong>Alignment</strong> Chaotic Good</p>\n<p><strong>Perception</strong> [[/r {1d20+12}]]{+12 Perception}; imprecise hearing 60 feet</p>\n<p><strong>Communication</strong> telepathic images (Common), 60 feet</p>\n<p><strong>Int</strong> +2, <strong>Wis</strong> +2, <strong>Cha</strong> +4</p>\n<p><strong>Will</strong> [[/r {1d20+15}]]{+15 Will}</p>\n<hr />\n<p>Working together, Lumna and the crystal key form an intelligent magic item. This item begins the adventure with 10 charges.</p>\n<p><strong>Activate</strong> <span class=\"pf2-icon\">R</span> envision; <strong>Cost</strong> 1 charge</p>\n<p><strong>Trigger</strong> A creature within 60 feet of the key takes electricity or mental damage</p>\n<p><strong>Effect</strong> Lumna grants the target resistance 10 to the triggering damage.</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"\\&quot;pf2-icon\\&quot; pf2-icon\">1</span> command (mental); <strong>Cost</strong> 1 charge</p>\n<p><strong>Effect</strong> Lumna floods the brain of a creature within 30 feet with painful telepathic images, inflicting [[/r {3d8}[mental]]]{3d8 mental damage} with a @Check[type:will|dc:24|basic:true] ([[/r {4d8}[mental]]]{4d8 mental damage}, DC 27 for Levels 9-10). The key can spend an additional action and 1 more charge to double the amount of damage or change the area to a @Template[type:cone|distance:15], but then can't use this activation again for [[/r 1d4 #rounds]]{1d4 rounds}.</p>\n<hr />\n<p><strong>Activate</strong> (10 minutes) command, Interact (exploration); <strong>Cost</strong> 5 charges</p>\n<p><strong>Effect</strong> Lumna briefly transforms all allies within 60 feet into energy, renewing them as if they had spent a full day resting. These allies regain Hit Points and recover from conditions as if they had 24 hours of rest. They can make their daily preparations again, with access to all their spell slots and any other daily abilities, even if they already used or expended them. Daily preparations happen as part of this activation, even if it would normally require an hour of study, prayer, or communion.</p>\n<hr />\n<p><strong>Activate</strong> (1 minute) command, Interact</p>\n<p><strong>Frequency</strong> once per day; <strong>Cost</strong> 2 charges</p>\n<p><strong>Effect</strong> Lumna returns the key and all living creatures within 60 feet to the Ardis Scholars' manor.</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Alignment</strong> Chaotic Good</p>\n<p><strong>Perception</strong> [[/r {1d20+12}]]{+12 Perception}; imprecise hearing 60 feet</p>\n<p><strong>Communication</strong> telepathic images (Common), 60 feet</p>\n<p><strong>Int</strong> +2, <strong>Wis</strong> +2, <strong>Cha</strong> +4</p>\n<p><strong>Will</strong> [[/r {1d20+15}]]{+15 Will}</p>\n<hr />\n<p>Working together, Lumna and the crystal key form an intelligent magic item. This item begins the adventure with 10 charges.</p>\n<p><strong>Activate</strong> <span class=\"pf2-icon\">R</span> envision; <strong>Cost</strong> 1 charge</p>\n<p><strong>Trigger</strong> A creature within 60 feet of the key takes electricity or mental damage</p>\n<p><strong>Effect</strong> Lumna grants the target resistance 10 to the triggering damage.</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"pf2-icon\">1</span> command (mental); <strong>Cost</strong> 1 charge</p>\n<p><strong>Effect</strong> Lumna floods the brain of a creature within 30 feet with painful telepathic images, inflicting [[/r {3d8}[mental]]]{3d8 mental damage} with a @Check[type:will|dc:24|basic:true] ([[/r {4d8}[mental]]]{4d8 mental damage}, @Check[type:will|dc:27|basic:true]  for Levels 9-10). The key can spend an additional action and 1 more charge to double the amount of damage or change the area to a @Template[type:cone|distance:15], but then can't use this activation again for [[/r 1d4 #rounds]]{1d4 rounds}.</p>\n<hr />\n<p><strong>Activate</strong> (10 minutes) command, Interact (exploration); <strong>Cost</strong> 5 charges</p>\n<p><strong>Effect</strong> Lumna briefly transforms all allies within 60 feet into energy, renewing them as if they had spent a full day resting. These allies regain Hit Points and recover from conditions as if they had 24 hours of rest. They can make their daily preparations again, with access to all their spell slots and any other daily abilities, even if they already used or expended them. Daily preparations happen as part of this activation, even if it would normally require an hour of study, prayer, or communion.</p>\n<hr />\n<p><strong>Activate</strong> (1 minute) command, Interact</p>\n<p><strong>Frequency</strong> once per day; <strong>Cost</strong> 2 charges</p>\n<p><strong>Effect</strong> Lumna returns the key and all living creatures within 60 feet to the Ardis Scholars' manor.</p>"
```